### PR TITLE
Make testnets optional in guide, remove bootnodes

### LINKED ISSUE #1914: Remove `--bootnodes` recommendation from docs for the mainnet Release
Bootnodes were needed for Ropsten since ethereum client syncing of that chain was always a pain (https://github.com/raiden-network/raiden/issues/1901). For mainnet this is not required and the bootnodes for ropsten would not work so they need to be removed before we launch in the mainnet.

### PR DESCRIPTION
**Pull request by palango**
Fixes #1914

### CODE REVIEW

File: docs/overview_and_guide.rst [hunk 1]
```diff
@@ -137,9 +137,12 @@ Firing it up
 Using geth
 **********
 
-Run the Ethereum client and let it sync with the Ropsten testnet::
+Run the Ethereum client and let it sync::
 
-    geth --testnet --fast --rpc --rpcapi eth,net,web3 --bootnodes "enode://20c9ad97c081d63397d7b685a412227a40e23c8bdc6688c6f37e97cfbc22d2b4d1db1510d8f61e6a8866ad7f0e17c02b14182d37ea7c3c8b9c2683aeb6b733a1@52.169.14.227:30303,enode://6ce05930c72abc632c58e2e4324f7c7ea478cec0ed4fa2528982cf34483094e9cbc9216e7aa349691242576d552a2a56aaeae426c5303ded677ce455ba1acd9d@13.84.180.240:30303"
+    geth --fast --rpc --rpcapi eth,net,web3
+
+.. note::
```
Comment (LefterisJP) [inline docs/overview_and_guide.rst]: perhaps also add a note on private nets with the chainid argument

File: docs/overview_and_guide.rst [hunk 2]
```diff
@@ -152,9 +155,12 @@ Then launch Raiden with the default testnet keystore path::
...
 ************
 
-Run the client and let it sync with the Ropsten testnet::
+Run the client and let it sync::
+
+    parity
 
-    parity --chain ropsten --bootnodes "enode://20c9ad97c081d63397d7b685a412227a40e23c8bdc6688c6f37e97cfbc22d2b4d1db1510d8f61e6a8866ad7f0e17c02b14182d37ea7c3c8b9c2683aeb6b733a1@52.169.14.227:30303,enode://6ce05930c72abc632c58e2e4324f7c7ea478cec0ed4fa2528982cf34483094e9cbc9216e7aa349691242576d552a2a56aaeae426c5303ded677ce455ba1acd9d@13.84.180.240:30303"
+.. note::
+    When you want to use a testnet add the ``--chain ropsten`` or ``--chain kovan`` flags.
```
Comment (LefterisJP) [inline docs/overview_and_guide.rst]: same as above

File: docs/overview_and_guide.rst [hunk 3]
```diff
@@ -152,9 +155,12 @@ Then launch Raiden with the default testnet keystore path::
 Using parity
 ************
 
-Run the client and let it sync with the Ropsten testnet::
+Run the client and let it sync::
+
+    parity
```
Comment (LefterisJP) [inline docs/overview_and_guide.rst]: we don't need any additional arguments for parity? I.e. perhaps warn about warp sync here?

Comment (palango) [reply docs/overview_and_guide.rst]: There is a warning below, but I'll add the `--no-warp` to the command.

**Pull request merged**